operations.js

```
@@ -0,0 +1,42 @@
+// Returns addition of two or n numbers
+exports.addition = function (args) {
+    var result = 0;
+    args.forEach(element => {
+        result += element;
+    });
+    return result;
+};
+
+// Returns difference of two numbers
+exports.subtraction = function (args) {
+    if (args.length > 2) {
+        return "can take only 2 arguments";
```

Should throw an error instead of return